ucfcs/GrowthPlus
# Ruth/quiz flashcard content

Added level 1 flashcard and quiz content.
Restructured quiz schema to have quiz content. 
Reverted styling from previous pr. 

To Test:
- Wipe app data and make children
- Check realm to see if quiz and flashcard content are populated 

Closes #77
